docs/src/markdown/ListItemDoc.mdx

```diff
@@ -1,7 +1,12 @@
+import { ImageGrid } from '../images/Image.tsx';
```

Comment:
Should image grid live in the images folder since it isn't an image or should we have this in some kind of utility folder/file?